File: 1/tugas1.java
import java.util.Scanner;
class tugas1 {
    public static void main(String args[]) {
    char[] a = {'x','y','z','h','g'}; 
    int[][] x = {{0,1}, {2,3}, {4,5}}; 

    Scanner input = new Scanner(System.in);
    System.out.print("Nilai : ");
    int n = input.nextInt();                //Input
    System.out.println("Nilainya : " + n);  //output
    
    if (n % 2 == 0) {
        System.out.println("Bilangan Genap");
    } else {
        System.out.println("Bilangan Ganjil");
    }

    System.out.println("ARRAY 1D");
    for(int i=0; i<5; i++){        //perulangan For & array 1D
        System.out.println("Index " + i + " = " + a[i]);            
    }
    int f = 0;
    while (f < 5) { //While
        System.out.println("HALLO AUTHOR");
        f++;
    }
    int e = 0;
    do { //Do while
        System.out.println("WELCOME LORD");
    e++;
    }
    while (e < 5);

    System.out.println("ARRAY 2D");
    for(int i=0;i<3;i++)
	{
		for(int j=0;j<2;j++)
		{
			System.out.print(x[i][j] + "\t");
		}
        System.out.println();
	}
    }
}

File: 1/tugas1.cpp
#include <iostream>
using namespace std;
int main(){
    char a[5]={'x','y','z','h','g'};   int b;
    int x[3][2] = {{0,1}, {2,3}, {4,5}};
    cout<<"Nilai : "; cin>>b;           //input
    
    cout<<"Array 1D"<<endl;
    for(int i=0; i<5; i++){        //perulangan For & array 1D
        cout<<"index "<<i<<" = "<<a[i]<<endl;             //output
    }
    if (b%2==0){ //For percabngan
        cout<<b<<" bilangan genap"<<endl;
    }else{
        cout<<b<<" bilangan ganjil"<<endl;
    }
    cout<<endl<<endl<< "PERULANGAN"<<endl;
    int f = 0;
    while (f < 5) { //While
        cout << "helloworld!"<<endl;
        f++;
    }
    int e = 0;
    do { //Do while
        cout << "HellowDunia" <<endl;
    e++;
    }
    while (e < 5);

	cout<<endl<<endl<< "Array 2D"<<endl;
    for(int i=0;i<3;i++)
	{
		for(int j=0;j<2;j++)
		{
			cout<<x[i][j]<<"\t";
		}
		cout<<endl;
	}
}